options/locale/locale_en-US.ini

```diff
@@ -1618,6 +1618,7 @@ pulls.reopened_at = `reopened this pull request <a id="%[1]s" href="#%[1]s">%[2]
 pulls.merge_instruction_hint = `You can also view <a class="show-instruction">command line instructions</a>.`
 pulls.merge_instruction_step1_desc = From your project repository, check out a new branch and test the changes.
 pulls.merge_instruction_step2_desc = Merge the changes and update on Gitea.
+pulls.clear_merge_message = Clear merge message
 
```

Comment:
I think this could do with a tooltip to explain that it removes the message but not the trailers.

Reply:
I tried adding a tooltip but it looks like this does not work within vue components. I assume because tooltips are handled by jquery/foomantic js which does not talk to vue js. Looks like [the file diff list](https://github.com/go-gitea/gitea/blob/main/web_src/js/components/DiffFileList.vue#L36) uses a workaround to make this load automatically in the vue component. This is probably something that should be cleaned up at some point so that it uses tippy's own vue plugin with proper support for directives.

I see two options on how to solve this in this PR:

1. Copy the same workaround from the diff file list component
2. Don't add a tooltip - maybe we could just show the explainer text below the button in a light gray?

I'd like to go with the second option.

Reply:
wouldn't a plain old `title=` attribute work?

Reply:
I've now implemented this:

![image](https://user-images.githubusercontent.com/13721712/199099365-a809c9b1-64ad-4ca1-b195-39ab0450be41.png)

Reply:
no probs

Reply:
> wouldn't a plain old `title=` attribute work?

Probably, yes. I didn't think about that, was focused to much on the tooltip that's used everywhere else :)